ols/app/endpoints/feedback.py

```diff
@@ -1,19 +1,58 @@
 """Handler for REST API call to provide user feedback."""
 
+import json
 import logging
+from pathlib import Path
 from typing import Any
 
 from fastapi import APIRouter, Depends
 
 from ols.app.models.models import FeedbackRequest, FeedbackResponse
+from ols.utils import config
 from ols.utils.auth_dependency import auth_dependency
 
 logger = logging.getLogger(__name__)
 router = APIRouter(prefix="/feedback", tags=["feedback"])
 
 
+def store_feedback(feedback_request: FeedbackRequest) -> None:
+    """Store feedback in the local filesystem.
+
+    Args:
+        feedback_request: The request containing feedback information.
+    """
+    # ensures storage path exists
+    storage_path = Path(config.ols_config.feedback_storage_location)
+    if not storage_path.exists():
+        storage_path.mkdir(parents=True)
+
+    # stores feedback in a file named after the conversation id
+    feedback_file_path = storage_path / f"{feedback_request.conversation_id}.json"
+    with open(feedback_file_path, "w", encoding="utf-8") as feedback_file:
+        json.dump(feedback_request.feedback_object, feedback_file)
```

Comment:
Do we need to store the conversation_id in the json too?

Reply:
I guess so, as the filename is not strictly part in the context.